src/main/groovy/com/bmuschko/gradle/java2html/ConvertCodeTask.groovy

```diff
@@ -30,23 +30,23 @@ import org.gradle.api.tasks.TaskAction
  */
 class ConvertCodeTask extends DefaultTask {
```

Comment:
While we are at it...should we also make the tasks `@Cacheable` tasks? That would also mean that we should normalize the file references with `@PathSensitive(RELATIVE)`.

Reply:
OK